ClearKeep/Scenes/Search/SearchWorker.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  SearchWorker.swift
+//  ClearKeep
+//
+//  Created by MinhDev on 05/04/2022.
+//
+
+import Networking
+import Model
+import ChatSecure
+import RealmSwift
+
+
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/fe9f474e9dacdb67166a3b18715e917932515a0f/ClearKeep/Scenes/Search/SearchWorker.swift#L13"></span>Limit vertical whitespace to a single empty line. Currently 2.<br />
<code>vertical_whitespace</code> <code>SearchWorker.swift:13</code></td>
    </tr>
  </tbody>
</table>